src/BlockEntities/JukeboxEntity.cpp

```diff
@@ -23,6 +26,21 @@ cJukeboxEntity::cJukeboxEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Ve
 
 cJukeboxEntity::~cJukeboxEntity()
 {
+	if (m_World)
+	{
+		// Stop playing music when destroyed by any means
+		m_World->BroadcastSoundParticleEffect(EffectID::SFX_RANDOM_PLAY_MUSIC_DISC, GetPos(), 0);
```

Comment:
Why is this needed? When is a block entity removed from the world without first calling `Destroy`?

Reply:
E.g. WorldEdit.